app/src/main/java/lc/wise/finenancer/presentation/assets/list/AssetsListFragment.kt

```diff
@@ -0,0 +1,9 @@
+package lc.wise.finenancer.presentation.assets.list
+
+import lc.wise.finenancer.databinding.FragmentAssetsListBinding
+import lc.wise.finenancer.presentation.BaseFragment
+
+class AssetsListFragment : BaseFragment<FragmentAssetsListBinding>() {
+    override fun getViewBinding(): FragmentAssetsListBinding = FragmentAssetsListBinding
+        .inflate(layoutInflater)
```

Comment:
```suggestion
    override fun getViewBinding() = FragmentAssetsListBinding.inflate(layoutInflater)
```
Наверное так немного получше выглядит